docker_push.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-docker build . -t "064061306967.dkr.ecr.us-west-2.amazonaws.com/acceptto/local-reverse-geocoder:1.0.0"
-docker push 064061306967.dkr.ecr.us-west-2.amazonaws.com/acceptto/local-reverse-geocoder:1.0.0
+# docker build . -t "064061306967.dkr.ecr.us-west-2.amazonaws.com/acceptto/local-reverse-geocoder:1.0.0"
+# docker push 064061306967.dkr.ecr.us-west-2.amazonaws.com/acceptto/local-reverse-geocoder:1.0.0
+
+docker buildx build --platform linux/amd64,linux/arm64/v8 --push -t 064061306967.dkr.ecr.us-west-2.amazonaws.com/acceptto/local-reverse-geocoder:1.2.0 .
```

Comment:
It might be good to pull the platforms out into a variable at the top for easier scanning if we ever need to change it in the future.
```suggestion
PLATFORMS=linux/amd64,linux/arm64/v8

docker buildx build --platform $PLATFORMS --push -t 064061306967.dkr.ecr.us-west-2.amazonaws.com/acceptto/local-reverse-geocoder:1.2.0 .
```

Reply:
It makes sense if we are using that value in multiple places but since this is only used in one place, it's already DRY. You can say the same thing for ECR repo URI, version, etc.